src/condor_tools/htcondor_cli/annex.py

```diff
@@ -10,7 +10,8 @@
 from htcondor_cli.noun import Noun
 from htcondor_cli.verb import Verb
 
-# Most of the annex create code is stored in a separate file.
+# Most of the annex add/create code is stored in a separate file.
+from htcondor_cli.annex_create import annex_add
 from htcondor_cli.annex_create import annex_create
```

Comment:
Your choice, just a little cleaner
```suggestion
from htcondor_cli.annex_create import annex_add, annex_create
```